src/libpriv/rpmostree-util.cxx

```diff
@@ -1172,3 +1172,23 @@ rpmostree_journal_error (GError *error)
 {
   sd_journal_print (LOG_WARNING, "%s", error->message);
 }
+
+void
+rpmostree_variant_be_to_native (GVariant **v)
```

Comment:
Nothing wrong with this, but it feels more obvious as an out parameter (i.e. function returns `GVariant*`) or alternatively we use C++ and it takes `GVariant &`?

Reply:
The idea with this one was to encapsulate the pointer swapping logic as well. Otherwise, it'd mostly be a really thin wrapper around `g_variant_byteswap`.